src/domain/usecases/CheckActionsAccessibleToCurrentUserUseCase.ts

```diff
@@ -0,0 +1,27 @@
+import _ from "lodash";
+import { ActionsPermissions } from "../entities/AppSettings";
+import { isSuperAdmin, User } from "../entities/User";
+import { getId } from "../entities/Ref";
+import { UserAction } from "../entities/UserAction";
+
+// Note for Reviewer: I made it sync to make it reactive, so I have the doubt if it shouldn't be a use case and just on the useHook
+// but as there is a CheckCurrentUserCanAccessSettingsUseCase it makes sense to have  this also as a usecase
+export class CheckActionsAccessibleToCurrentUserUseCase {
```

Comment:
I think this could be a hook since we already have all the information in the UI layer.